File: selfos-backend/Person.py
class Person:
    def __init__(self, firstname, lastname, imageurl, phone, email, streetAddress, aptNumber, city, state, zipcode):
        self.firstname = firstname
        self.lastname = lastname
        self.imageurl = imageurl
        self.phone = phone
        self.email = email
        self.streetAddress = streetAddress
        self.aptNumber = aptNumber
        self.city = city
        self.state = state
        self.zipcode = zipcode


    def __repr__(self):
        return self.firstname + self.lastname + self.imageurl + self.phone + self.email + self.streetAddress + \
               self.aptNumber + self.city + self.state + self.zipcode


    def jsonObject(self):
        personDict = dict()
        personDict["firstname"] = self.firstname
        personDict["lastname"] = self.lastname
        personDict["imageurl"] = self.imageurl
        personDict["phone"] = self.phone
        personDict["email"] = self.email
        personDict["streetAddress"] = self.streetAddress
        personDict["aptNumber"] = self.aptNumber
        personDict["city"] = self.city
        personDict["state"] = self.state
        personDict["zipcode"] = self.zipcode
        return personDict

File: selfos-ui/src/categories/People/Person.js
import React from 'react';

import "./Person.css";
import person from './contact.png';



class Person extends React.Component{

  constructor(props){
    super(props)
  }

  render(){
    // ensure we have image URL otherwise return the default imageUrl
    if(this.props.value.imageUrl == null || this.props.value.imageUrl == ""){
      return(
        <div class="person">
          <span class="span"><img class="img" src={person} /></span>
          <span>{this.props.value.firstName} {this.props.value.lastName}</span>
        </div>
      )
    }else{
      return(
        <div class="person">
          <span class="span"><img class="img" src={this.props.value.imageUrl} /></span>
          <span>{this.props.value.firstName} {this.props.value.lastName}</span>
        </div>
      )
    }
  }

  //           {home.firstName}
  //           {home.lastName}
  //           {home.city}
  //           {home.aptNumber}
  //           {home.email}
  //           {home.id}
  //           {home.imageUrl}
  //           {home.phone}
  //           {home.state}
  //           {home.streetAddress}
  //           {home.zipCode}
  //           {home.lastName}
  //           {home.lastName}
  //           </div>)}

}

export default Person;
